peachjam/migrations/0075_backfill_authors.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.18 on 2023-05-16 11:30
+
+from django.db import migrations
+
+
+def backfill_authors(apps, schema_editor):
+    CoreDocument = apps.get_model("peachjam", "CoreDocument")
```

Comment:
You need to do this separately for `LegalInstrument` and `GenericDocument` because `CoreDocument` doesn't have the field `author`.